.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Version
+[provide output of the `$ openshift version` command]
+[provide output of the `$ oc version` command and the version of the master]
```

Comment:
now these two lines look redundant.
"provide the version of the master by running $ openshift version on the master"
"provide the version of the client by running $ oc version on your client machine"